src/panels/config/zwave/zwave-network.ts

```diff
@@ -59,136 +56,136 @@ class ZwaveNetwork extends PolymerElement {
         <ha-card class="content">
           <div class="card-actions">
             <ha-call-service-button
-              hass="[[hass]]"
+              .hass="${this.hass}"
               domain="zwave"
               service="add_node_secure"
             >
               Add Node Secure
             </ha-call-service-button>
             <ha-service-description
-              hass="[[hass]]"
+              .hass="${this.hass}"
```

Comment:
Quotes are not necessary and it looks better without too 👍 

```suggestion
              .hass=${this.hass}
```